elements/pf-code-block/test/pf-code-block.spec.ts

```diff
@@ -28,12 +28,12 @@ const expandElement = html`
       connectionConfig:
       url: https://raw.githubusercontent.com/Azure-Samples/helm-charts/master/docs
     </script>
-    <pfe-clipboard slot="actions" copy-from="#expandable-code"></pfe-clipboard>
-  </pfe-code-block>
+    <pf-clipboard slot="actions" copy-from="#expandable-code"></pf-clipboard>
```

Comment:
`<pf-clipboard ...></pf-clipboard>` should be `<pf-clipboard-copy ...></pf-clipboard-copy>`